src/main/resources/db/changelog/userservice-dev-master.xml

```diff
@@ -30,4 +30,5 @@
 	<include file="db/changelog/changeset/0023_consultant_status/0023_changeSet.xml"/>
 	<include file="db/changelog/changeset/0024_consultant_walk_through/0024_changeSet.xml"/>
 	<include file="db/changelog/changeset/0025_consultant_agency_status/0025_changeSet.xml"/>
+	<include file="db/changelog/changeset/0026_session_main_topic/0026_changeSet.xml"/>
```

Comment:
See comment regarding version in previous PR, we need to increase the version ideally befor this is getting deployed